assets/src/scss/blocks/Spreadsheet.scss

```
@@ -0,0 +1,160 @@
+.block-spreadsheet {
+  display: inline-block;
+  margin-top: $sp-1;
+  margin-bottom: $sp-2;
+
+  @include large-and-up {
+    margin-top: $sp-2;
+    margin-bottom: $sp-4;
+  }
+}
+
+.table-wrapper {
+  overflow: auto;
+
+  max-height: 320px;
+  margin-top: $sp-3;
+
+  @include medium-and-up {
+    max-height: 520px;
+    margin-top: $sp-2;
+  }
+
+  @include large-and-up {
+    max-height: 575px;
+  }
+}
+
+table.spreadsheet-table {
+  min-width: 100%;
+  margin-bottom: 0;
+
+  @include mobile-only {
+    overflow: visible;
+  }
+
+  tr {
```

I mean something like this. What do you both think?

```suggestion
tr {
    background-color: var(--grey-100);
    
    &:nth-child(odd) {
       background-color: var(--grey-200);
    }
    
    td {
    color: var(--color-text-body);
    padding: 5px $sp-2x;
    font-size: var(--font-size-s--font-family-tertiary);
    white-space: pre-wrap;

    @include x-large-and-up {
      font-size: var(--font-size-s--font-family-tertiary);
    }

    .highlighted-text {
      font-weight: bold;
    }
  }
}
```